model/Service/Classes/SoftwareService.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SoftwareService
```

Comment:
Why not just "Service"?

Reply:
That was my original thought when I was writing this up.

After looking up different definitions of Software as a Service, I decided that we should qualify the name with "Software".

In the future, we could add other services that are not software if there ever is a use case.

That being said, I don't feel strongly about using SoftwareService over Service.